tests/foreman/cli/test_oscap.py

```diff
@@ -155,13 +188,18 @@ def test_negative_info_scap_content_viewer_role(self):
 
         :expectedresults: The info of the scap-content is not listed.
 
-        :caseautomation: notautomated
+        :caseautomation: automated
 
         :CaseImportance: Critical
         """
+        title = gen_string('alpha')
+        make_scapcontent({'title': title, 'scap-file': '/tmp/{0}'.format(
+                                           self.file_name)})
+        login, password = self.create_test_user_viewer_role()
+        result = Scapcontent.with_user(login, password).info({'name': title})
```

Comment:
Should be `.info({'title': title})` here, shouldn't it?

Reply:
This was a bug , fixed in latest snap.